Gopkg.toml

```diff
@@ -69,7 +69,15 @@
  source = "https://github.com/mandarjog/zipkin-go-opentracing.git"
 
 [[override]]
-    revision = "d4df91709b724174aaf8a957f3edac3573be354e"
-    name = "github.com/apache/thrift"
+  revision = "d4df91709b724174aaf8a957f3edac3573be354e"
+  name = "github.com/apache/thrift"
+
+[[constraint]]
+  name = "istio.io/api"
+  branch = "master"
```

Comment:
why is this fixed to master? We need to pin to the same sha used by WORKSPACE. Otherwise, this would constantly fail when someone updates API but has not yet updated Istio with respective changes.

Reply:
It is the other way around. One first updates the api, then wants to pick up the change in istio.

Reply:
This fixes the build problem a few people complained about on slack. There was a new field added in api, used in istio, but not picked up by go dep, most likely because go dep already have a .lock file. This will force to latest.

If you pin it to a fixed version, it will have to be moved every time a new change is made; more hassle, more things being forgotten.

Reply:
And btw, it's time to require backward compatible changes: this means no change in istio/api should remove a field and cause istio/istio to fail.

Reply:
Isn't that the model we have been following? update SHA in istio/istio whenever api changes? I don't mind using master, but soon there will be some change that breaks istio/istio circle ci because some api restructuring is not reflected. I will revert to @guptasu for final call, since he spent the day trying to fix circle ci.

Reply:
Re API compatibility: we're at alpha for all APIs, there's no forwards- or backwards- compatibility that we promised.